cfpb/erap-data-processing
# Add Denver to county map

Add Denver to county map to handle new city addition to data.

## Additions

- Denver in county map


## Testing

1. Check out branch and run script against new data
2.  Verify that Denver has correct county data in output file
